arangodb-net-standard.Test/GraphApi/GraphApiClientTest.cs

```diff
@@ -81,7 +83,7 @@ public async Task DeleteGraphAsync_ShouldThrow_WhenNotFound()
             });
 
             Assert.Equal(HttpStatusCode.NotFound, exception.ApiError.Code);
-            Assert.Equal(1924, exception.ApiError.ErrorNum); // GRAPH_NOT_FOUND
+            Assert.Equal((int)ERROR_GRAPH_NOT_FOUND, exception.ApiError.ErrorNum);
```

Comment:
🌱 Perhaps int constants would be better instead of enum values.

```
public static ArangoDBErrors
{
  public const int ERROR_GRAPH_NOT_FOUND = 1924;
}
```

It would be a breaking change in theory, but I can't imagine consumers doing anything else than int comparison against `ApiErrorResponse.ErrorNum`.